interface/src/app/signature/enriched_genesets.tpl.html

```diff
@@ -12,13 +17,30 @@
     </tr>
   </thead>
 
-  <tr ng-repeat="geneset in enrichedGenesets">
+  <tr ng-repeat="geneset in enrichedGenesets | limitTo: numGenesetsShown">
```

Comment:
Totally, thanks! They do help implementing stuff like this and making it concise and neat.